tests/PHPUnit/Framework/Mock/TestConfig.php

```diff
@@ -73,4 +74,60 @@ public function forceSave()
             parent::forceSave();
         }
     }
+
```

Comment:
there is no need for a docblock: the method name is explicit enough, and the parameter has a type-hint.